src/tests/parse.test.ts

```diff
@@ -0,0 +1,22 @@
+import assert from "assert/strict";
+import { BigNumber } from "bignumber.js";
+import parseAmount from "../helpers/parser.js";
+
+function testNormalBigAmount() {
+  const rawFungibleTokenAmount = "0x00000000000000000de0b6b3a7640000";
+  const bigNumberString = BigNumber(rawFungibleTokenAmount, 16).toFixed(0);
+  const amountPrettified = parseAmount(bigNumberString);
+  const expectedParsedAmount = "1.0";
+  assert.deepStrictEqual(amountPrettified, expectedParsedAmount);
+}
+
+function testScientificBigAmount() {
+  const rawFungibleTokenAmount = "0x000000000000182a697403a1b0cf0ad2";
```

Comment:
Just for reference this value was extracted from Mainnet Indexer under [this tx](https://avail.subscan.io/extrinsic/0x88d21d6c9dc753d4d05abc50ffb355d4cf5dbc7361c1dd58e7ee5c8bb5b6770a)

GQL query:

```gql
query {
  extrinsics(
    filter: {
      call: { equalTo: "execute" }
      success: { equalTo: true }
      blockHeight: { equalTo: "144917" }
    }
    first: 2
    orderBy: BLOCK_HEIGHT_ASC
  ) {
    nodes {
      id
      txHash
      module
      call
      blockHeight
      success
      isSigned
      extrinsicIndex
      hash
      timestamp
      signer
      signature
      fees
      nonce
      argsName
      argsValue
      nbEvents
    }
  }
}
```